graphql/concrete_notification/fields.ts

```diff
@@ -99,3 +101,53 @@ export class ExtendedFieldsConcreteNotificationResolver {
         return Object.values(byCampaign);
     }
 }
+
+@ObjectType()
+class ConcreteNotificationFormatted {
+    @Field((type) => ID)
+    id: number;
+    @Field()
+    headline: string;
+    @Field()
+    body: string;
+    @Field()
+    notificationClass: string;
```

Comment:
What is a notification class? And why is it on the concrete notification and not the notification?

Reply:
this should reflect notification type, like "reminder", "match" ... which would be mapped to different icons in frontend. I thought "notificationClass" might be good name, but I am open to all suggestions

Reply:
I think you are right and this information should come from Notification. Not sure if it should change the query to get this over notification or would it be an option to "transfer" this field directly to concrete_notification